test/Microsoft.ML.Tests/TrainerEstimators/OnlineLinearTests.cs

```diff
@@ -36,16 +36,14 @@ public void OnlineLinearWorkout()
                 .Append(r => (r.Label, Features: r.Features.Normalize()));
 
             var binaryTrainData = binaryPipe.Fit(binaryData).Transform(binaryData).AsDynamic;
-            var apTrainer = new AveragedPerceptronTrainer(ML, "Label", "Features", lossFunction: new HingeLoss(), advancedSettings: s =>
```

Comment:
>lossFunction: new HingeLoss() [](start = 83, length = 29)

set this inside Options, so we keep the test the same.

Reply:
Created issue #2174.  We need to fix how LossFunction is defined for AvgPerceptron and OnlineGradientDescent

Unfortunately, i cant add a "skipped" test  because  its a compile time error  (not a run-time issue). 

---
In reply to: [248545068](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248545068) [](ancestors = 248545068)